theano/sandbox/cuda/nvcc_compiler.py

```diff
@@ -404,7 +404,8 @@ def compile_str(
         elif config.cmodule.compilation_warning and nvcc_stdout:
             print(nvcc_stdout)
 
-        if nvcc_stdout:
+        # On Windows, nvcc print useless stuff by default
+        if sys.platform != 'win32' and nvcc_stdout:
             # this doesn't happen to my knowledge
             print("DEBUG: nvcc STDOUT", nvcc_stdout, file=sys.stderr)
```

Comment:
This is the old back-end. I don't want to spend more time on it.

On Wed, Apr 20, 2016 at 12:04 AM, Pascal Lamblin <notifications@github.com>
wrote:

> In theano/sandbox/cuda/nvcc_compiler.py
> <https://github.com/Theano/Theano/pull/4406#discussion_r60347957>:
>
> > @@ -404,7 +404,8 @@ def compile_str(
> >          elif config.cmodule.compilation_warning and nvcc_stdout:
> >              print(nvcc_stdout)
> >
> > -        if nvcc_stdout:
> > +        # On Windows, nvcc print useless stuff by default
> > +        if sys.platform != 'win32' and nvcc_stdout:
> >              # this doesn't happen to my knowledge
> >              print("DEBUG: nvcc STDOUT", nvcc_stdout, file=sys.stderr)
>
> I think we can just remove those lines, or call _logger.info() instead of
> print. There is no need to check the platform if the goal is for the
> message not to be displayed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/4406/files/191b88b36c7a78bd836fdd3cc9a402f137576dc1#r60347957>
>